templates/customers/addresses.liquid

```diff
@@ -29,10 +29,10 @@
     {% comment %}
       Add address form, hidden by default
     {% endcomment %}
-    <div id="AddAddress" class="form-vertical" style="display: none;">
-      {% form 'customer_address', customer.new_address %}
-
+    {% form 'customer_address', customer.new_address %}
+      <div id="AddAddress" class="form-vertical" {% unless form.errors %}style="display:none;"{% endunless %}>
```

Comment:
Hm, with the `form` wrapped around this we'll always have the element visible on the page (though empty). I can't think of another way (without JS) so as long as this doesn't effect the layout, 👍